.gitignore

```diff
@@ -133,3 +133,4 @@ dmypy.json
 
 # PyCharm config files
 .idea/
+bsmetadata/vendor/dateutil
```

Comment:
What's this line for?  I don't think it's necessary.
I am not sure if this would cause a problem or not.

Reply:
✅ just my local workaround for working with submodules in visual code and not necessary for the public version